### PR TITLE
Add sidebar

### PR DESCRIPTION
**Pull request by lazarmitic**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/sidebar/Sidebar.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.tc__sidebar-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 25px;
+  padding-right: 15px;
+  border-radius: 5px 0px 0px 5px;
+  box-shadow: 2px 0px 4px rgba(0, 0, 0, 0.25);
+  background: rgba(255, 255, 255, 0.7);
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/sidebar/Sidebar.scss:9]: we should add this background to `constants.scss`, I think this value is already used somewhere else

File: packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      <TokenPairIcon parentTicker="ETH" childTicker="stETH" />
+      <Spacer size={5} />
+      <Typography variant="h4">
+        {/* TODO Update ticker based on selected pool */}
+        {'stETH'} Pool
+      </Typography>
+      <Spacer size={10} />
+      <Typography variant="body-text" color="link">
+        {/* TODO Show selected pool address here - it should be clickable, and it should open pool etherscan link in new tab */}
+        {'0x123...50E'}
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx:30]: I'd add the `<a>` tag then

File: packages/tempus-client_v2/src/components/sidebar/tokenPairIcon/TokenPairIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  parentTicker: Ticker;
+  childTicker: Ticker;
+}
+
+const TokenPairIcon: FC<TokenPairIconProps> = props => {
+  const { parentTicker, childTicker } = props;
+
+  return (
+    <div className="tc__tokenPairIcon-parent">
+      <TokenIcon ticker={parentTicker} width={40} height={40} />
```
Comment (danieledamiani) [inline packages/tempus-client_v2/src/components/sidebar/tokenPairIcon/TokenPairIcon.tsx:17]: are our token SVGs ok when rendered at 40x40 pixels

**Review by danieledamiani — CHANGES_REQUESTED** [review]
A few comments

File: packages/tempus-client_v2/src/components/sidebar/tokenPairIcon/TokenPairIcon.tsx (hunk 3 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/sidebar/tokenPairIcon/TokenPairIcon.tsx:17]: Yes, I checked them and they look fine.

File: packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx (hunk 2 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx:30]: I don't have link to put into <a> tag, and there is a warning for an a tag without link.

File: packages/tempus-client_v2/src/components/sidebar/Sidebar.scss (hunk 1 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/sidebar/Sidebar.scss:9]: fixed.

File: packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx (hunk 2 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx:30]: `https:\\anysite.com`?

**Review by danieledamiani — APPROVED** [review]

File: packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx (hunk 2 above)
Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx:30]: well yeah, I can add placeholder link.

Comment (lazarmitic) [reply packages/tempus-client_v2/src/components/sidebar/Sidebar.tsx:30]: fixed

**Pull request merged**